scripts/decals/build_galaxy10_sample.py

```diff
@@ -0,0 +1,125 @@
+import logging
+from multiprocessing import Pool
+from io import BytesIO
+import os
+from pathlib import Path
+from functools import partial
+
+import numpy as np
+import tqdm
+import pandas as pd
+import requests
+import h5py
+from astropy.io import fits
+
+
+from create_images_from_cutout_server import get_download_url
+
+
+
+
+def run_download(df, overwrite, n_processes=4):
+    # download_image(train_df.iloc[0])
+
+    pool = Pool(processes=n_processes)
+    rows = [row for _, row in df.iterrows()]
+
+    retry_wrapper_partial = partial(retry_wrapper, download_image, overwrite=overwrite)
+
+    # retry_wrapper_partial(rows[0])
+
+    logging.info('Beginning download')
+    res = pool.imap_unordered(retry_wrapper_partial, rows)
+    list(res)  # iterate
+    logging.info('Finished download')
+
+
+def pack_to_hdf5(df: pd.DataFrame, save_loc: Path):
+    # save to hdf5
+    with h5py.File(save_loc, 'w') as f:
+ 
+        for col in ['ra', 'dec', 'label']:
+            f.create_dataset(col, data=df[col].values)
+
+        f.create_dataset('image_array', (len(df), 3, 224, 224))  # no values yet
```

Comment:
the image size is hard coded - perhaps try to make it clear somewhere in the code with a global variable and replace all numerical values by the global variable name